packages/ui-core/src/components/BubbleHint/BubbleHint.tsx

```diff
@@ -80,6 +83,16 @@ class BubbleHint extends React.Component<Partial<IBubbleHintProps>, IBubbleHintS
         document.addEventListener(this.touchClick, this.handleClickOutside);
     }
 
+    componentDidUpdate(_prevProps: Readonly<Partial<IBubbleHintProps>>, prevState: Readonly<IBubbleHintState>): void {
```

Comment:
```suggestion
    componentDidUpdate(prevProps: Readonly<IBubbleHintProps>, prevState: Readonly<IBubbleHintState>): void {
```
У компонента пропсы определяются как `Partial<IBubbleHintProps>`, при этом в самом интерфейсе IBubbleHintProps какие-то поля опциональные, а какие-то обязательные. В общем непонятно зачем это. Единственное, где это нужно - в тестах, но сам компонент это по идее не должно затрагивать. Предлагаю поменять.